exercises/simple-cipher/simple_cipher_test.exs

```diff
@@ -0,0 +1,117 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("simple_cipher.exs", __DIR__)
+end
+
+ExUnit.start
+ExUnit.configure trace: true, exclude: :pending
+
+defmodule SimpleCipherTest do
+  use ExUnit.Case
+
+  describe "random key generation" do
+    #@tag :pending
+    test "random key is all lowercase letters" do
+      key = SimpleCipher.key
+
+      assert key != nil
```

Comment:
I didn't have this line at first, because the next line does indeed fail, but the test failure was an actual exception when trying to pass nil to the regex match, instead of a test failure, so I thought it'd be less confusing to users this way.

Actually, now that I've had a day to think about it, I don't like the key generation tests at all. They sort of presuppose a language where you're going to store the key, not pass it in to other functions. It doesn't fit in with the rest of the tests at all, because they have a known key, so they can test for known outputs. I could seed the RNG, but then what's the point? If it's supposed to be 100 random characters, how do you test that randomness?

The result is never actually used anywhere else in the tests, so unless you object, I think I'm going to get rid of those tests and the function in general.